app/serializers/activitypub/actor_serializer.rb

```diff
@@ -186,7 +186,7 @@ def href
     end
 
     def name
-      "##{object.name}"
+      object.formatted_name
```

Comment:
`Tag#formatted_name` uses the `#display_name` while this method used to use the `#name`. Those can be different (`name` being a normalized version of `display_name as per #18795).

So this here would actually change the behavior. (Of course I am in no way certain the old behavior was actually the intended one or an oversight in the linked PR :confused:).

Reply:
Hmm, good point! I'm not sure what the intended result should be here?

No tests began failing with this change 

cc @ClearlyClaire

Reply:
That is a very good point. I suspect we might want `formatted_name` indeed, but I'd prefer that to be in another PR, as this does not seem related to the change at hand.